lang/es.xml

```diff
@@ -3,29 +3,45 @@
 <lang>
 	<main_menubar>
 		<file name="Archivo">
-			<open_pref_path>Abrir la Carpeta Pref</open_pref_path>
-			<install_firmware>Instalar Firmware</install_firmware>
-			<install_pkg>Instalar un .pkg</install_pkg>
-			<install_zip>Instalar un .zip, .vpk</install_zip>
+			<open_pref_path>Abrir la carpeta de preferencias</open_pref_path>
```

Comment:
totaly no view, no change, if first leter is caps is for good reson[](url)

Reply:
No need, is Just Pref Path, no any other things, no add more info

Reply:
in french
![image](https://user-images.githubusercontent.com/5261759/176547573-3810e661-0a6c-45dd-97d7-7c5eaa8e0936.png)

Reply:
Pref can be uppercased, but folder should not be uppercased. Is that okay?

Reply:
more look all exept one, the is caps,

Reply:
That is not proper Spanish. You are trying to apply French uppercasing/capitalization rules to a different language that does not work that way.

Reply:
no, is juste used for all languague, english too, all word is in caps for first letter, is US rulle of style like for button

Reply:
I am telling you that English Title Uppercasing rules do not apply in Spanish, according to the RAE. Menu tabs, menu headers and menu elements should not use Title Uppercasing in Spanish.